libevmasm/AssemblyItem.h

```diff
@@ -160,6 +160,7 @@ class AssemblyItem
 	langutil::SourceLocation const& location() const { return m_location; }
 
 	void setJumpType(JumpType _jumpType) { m_jumpType = _jumpType; }
+	void setJumpTypeFromString(std::string const& _jumpType);
```

Comment:
These two have essentially the same effect (setting jump type). I'd use an overload.
```suggestion
	void setJumpType(std::string const& _jumpType);
```